ArduPlane/ArduPlane.cpp

```diff
@@ -695,7 +695,7 @@ void Plane::get_osd_roll_pitch_rad(float &roll, float &pitch) const
 {
    pitch = ahrs.pitch;
    roll = ahrs.roll;
-   if (!quadplane.show_vtol_view()) {  // correct for TRIM_PITCH_CD
+   if (!quadplane.show_vtol_view() && g2.flight_options & FlightOptions::OSD_DISPLAY_TRIM_PITCH_CD) {  // correct for TRIM_PITCH_CD
       pitch -= g.pitch_trim_cd * 0.01 * DEG_TO_RAD;
    } else {
```

Comment:
```suggestion
   } else if (quadplane.show_vtol_view()){
```

Reply:
this is also needed for proper operation

Reply:
Ah! I missed the ! on the vtol view last night. Should this be an OR rather than AND?

Reply:
no....cases ::
FW view, no option bit set : remove TRIM_PITCH_CD
FW view, option bit set: fall thru with ahrs pitch, not removing TRIM_PITCH_CD
VTOL view: use VTOL view pitch and roll (which does not have TRIM_PITCH_CD and already includes Q_TRIM_PITCH in view pitch and in logged pitch)

Reply:
So the catchall else is not needed here?

Reply:
no.....if the first two IFs are not met, it falls thru to the end so pitch is what it was set at at the top and un-modified....if statements do not need else statements to be complete

Reply:
Cool. Just double checking. Haven't needed to jump into the quadplane vtol code yet. Fixed.